framework/cdv-gradle-config-defaults.json

```
@@ -7,7 +7,7 @@
     "AGP_VERSION": "7.2.1",
     "KOTLIN_VERSION": "1.5.21",
     "ANDROIDX_APP_COMPAT_VERSION": "1.4.2",
-    "ANDROIDX_WEBKIT_VERSION": "1.4.0",
+    "ANDROIDX_WEBKIT_VERSION": "1.5.0",
```

```suggestion
    "ANDROIDX_WEBKIT_VERSION": "1.6.0",
```
1.6.0 is now available